src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,5 @@
+$font-family: Roboto, Arial, Helvetica, sans-serif;
+$white-color: #fff;
+$blue-color: rgba(0, 172, 220, 1);
+$grey-color: rgba(97, 96, 112, 1);
+$color-black: rgba(6, 11, 53, 1);
```

Comment:
Consider using a consistent naming convention for color variables. You have both `$white-color` and `$color-black`. It would be better to stick to one format, either `$color-white` and `$color-black` or `$white-color` and `$black-color`, for better readability and maintainability.